sysl2/sysl/seqs.go

```diff
@@ -135,7 +159,7 @@ func DoConstructSequenceDiagrams(
 // DoGenerateSequenceDiagrams generate sequence diagrams for the given model
 // TODO: Remove nolint when code that uses stdout and stderr is added.
 //nolint:unparam
-func DoGenerateSequenceDiagrams(stdout, stderr io.Writer, args []string) error {
+func DoGenerateSequenceDiagrams(args []string) error {
```

Comment:
`//nolint:unparam` might not be required  now.

Reply:
Definitely. (Is there a linter-no-required linter?)